smartsim/_core/entrypoints/colocated.py

```diff
@@ -201,8 +202,17 @@ def main(
     # we generally want to catch all exceptions here as
     # if this process dies, the application will most likely fail
     try:
-        process = psutil.Popen(cmd, stdout=PIPE, stderr=STDOUT)
-        DBPID = process.pid
+        hostname = socket.gethostname()
+        filename = (
+            f"{hostname}.log"
+            if os.getenv("SMARTSIM_LOG_LEVEL") == "debug"
+            else "/dev/null"
```

Comment:
I also suggest "/dev/null" -> `os.devnull` see [here](https://docs.python.org/3/library/os.html#os.devnull) as a more portable (and nice looking) solution.

Reply:
Good catch! would h8 for this to fail for others